tabsforeditorswidget.cpp

```diff
@@ -183,6 +229,61 @@ void TabsForEditorsWidget::nextTabAction()
     m_tabWidget->setCurrentIndex(0);
   }
 }
+
+void TabsForEditorsWidget::handleTabRightButtonClick(int tabIndex, QPoint &position)
+{
+    QMenu contextMenu;
+
+    QList<QAction*> actionsList;
+
+    QAction* closeTabAction = new QAction(tr("Close Tab"), this);

```

Comment:
All the actions are created every time the context menu is requested. It would be nice to create the context menu once (at widget creation time), and later just show it.